platform/posix/transport/CMakeLists.txt

```diff
@@ -40,12 +40,47 @@ target_link_libraries( openssl_posix
                           # requires explicit linking.
                           ${CMAKE_DL_LIBS} )
 
+# Set path to corePKCS11 and it's third party libraries.
+set(COREPKCS11_LOCATION "${CMAKE_SOURCE_DIR}/libraries/standard/corePKCS11")
+set(CORE_PKCS11_3RDPARTY_LOCATION "${COREPKCS11_LOCATION}/source/dependency/3rdparty")
+
+# Include PKCS #11 library's source and header path variables.
+include( ${COREPKCS11_LOCATION}/pkcsFilePaths.cmake )
+
+list(APPEND PKCS_SOURCES
+    "${COREPKCS11_LOCATION}/source/portable/posix/core_pkcs11_pal.c"
+    "${CORE_PKCS11_3RDPARTY_LOCATION}/mbedtls_utils/mbedtls_utils.c"
+)
+
+# Create target for POSIX implementation of MbedTLS.
+add_library( mbedtls_pkcs11_posix
+                ${MBEDTLS_PKCS11_TRANSPORT_SOURCES}
+                ${PKCS_SOURCES} )
+
+target_link_libraries( mbedtls_pkcs11_posix
+                       PRIVATE
+                          mbedtls )
+
+target_include_directories(
+    mbedtls_pkcs11_posix
+    PUBLIC
+        ${COMMON_TRANSPORT_INCLUDE_PUBLIC_DIRS}
+        ${LOGGING_INCLUDE_DIRS}
+        ${TRANSPORT_INTERFACE_INCLUDE_DIR}
+        ${PKCS_INCLUDE_PUBLIC_DIRS}
+        ${CORE_PKCS11_3RDPARTY_LOCATION}/pkcs11
+        ${DEMOS_DIR}/pkcs11/common/include
```

Comment:
Why is inclusion of path from a demo folder required?

Reply:
That has the corePKCS11 config

Reply:
Can you add a comment about it?